README.md

```diff
@@ -1 +1,13 @@
-# example_navigation_app_ios
\ No newline at end of file
+# example_navigation_app_ios
+
+This example application is to be used as a reference for developers looking to use the [SmartDeviceLink](https://github.com/smartdevicelink/sdl_ios) platform with their mobile navigation app.
+
+You will need a [MapBox](https://www.mapbox.com/) access token and an app name & ID with the correct permissions if you plan to use this app in an SDL environment. To set the access token, app name, and app ID of this project, please follow the steps below.
+
+Setting App Keys
+
+1. Create a `keys.plist` file in the project.
+2. Create three key values named `AppName`, `AppID`, and `MGLMapboxAccessToken` of type `String`
+3. Set the values to the appropriate key.
+
+Note that, at minimum, the `MGLMapboxAccessToken` must be set in order to use the app.
```

Comment:
Additional Details:
1. Add more detail about why you need SDL permissions. Something like: If you are testing on production or test hardware it is very likely that this app will not work due to OEMs restricting permissions for video streaming apps. 
1. Add instructions on how to use the app:
   * Run `pod install` and open the `.xcworkspace` file.
    * How to configure an iAP or TCP connection. It looks like the app immediately starts looking for a connection?
    * Add instructions that the "render" and "stream" types need to be set before establishing a connection with the head unit. I think a lot of people will try to change the types while video is streaming and wonder why it isn't working. Maybe disable "type" selection while a connection is established?
    * It looks like the menu button on the phone app does something different than on the SDL app. Maybe add a short description of how the menu button works on each app.

Reply:
Updated the `README` to include the information above. Let me know if I need to add additional information into any of the sections.